### PR TITLE
Dark theme update

### PR DESCRIPTION
**Pull request by alexcherman**

### CODE REVIEW

**Comment by alexcherman** [comment]
> text-dark in the theme is .text-dark {
> color: #2a2c3f !important;
> }
> But this should be a light color in a dark theme. ( and a dark color in a light theme).

@ryestew  I thought we need to switch .text-dark and .text-light colors in dark theme. But if you want I can revert this change

File: src/app/components/plugin-manager-component.js [hunk 1]
```diff
@@ -196,7 +196,7 @@ class PluginManagerComponent extends ViewPlugin {
       <div id='pluginManager'>
         <header class="form-group ${css.pluginSearch} plugins-header py-3 px-4 border-bottom">
           <input onkeyup="${e => this.filterPlugins(e)}" class="${css.pluginSearchInput} form-control" placeholder="Search">
-          <button onclick="${_ => this.openLocalPlugin()}" class="${css.pluginSearchButton} btn bg-transparent text-dark border-0 mt-2 text-underline">
+          <button onclick="${_ => this.openLocalPlugin()}" class="${css.pluginSearchButton} btn bg-transparent text-dark border-0 mt-2 text-underline plugins-connect-btn">
```
Comment (ryestew) [inline src/app/components/plugin-manager-component.js]: I don't think you need to add this class - but to use the color of plugins-connect-btn as the color of text-dark.

**Comment by ryestew** [comment]
text-dark - is still too dark - please use #babbcc

**Comment by alexcherman** [comment]
> text-dark - is still too dark - please use #babbcc

ok, but "Connect to a Local Plugin" button has #A2A3BD color

**Comment by alexcherman** [comment]
@ryestew Updated text-dark color

**Comment by ryestew** [comment]
Actually there is an issue the alpha and the beta tags are different colors - alpha is missing a class that is in beta.

**Comment by alexcherman** [comment]
@ryestew I'll check it

**Pull request merged**